ci-operator/step-registry/cucushift/upgrade/arch-migration/cucushift-upgrade-arch-migration-commands.sh

```diff
@@ -65,6 +65,27 @@ function run_command_oc() {
     echo "${ret_val}"
 }
 
+# Wait upgrade to start, https://polarion.engineering.redhat.com/polarion/#/project/OSE/workitem?id=OCP-25473
+# Progressing=True
+# ReleaseAccepted=True
+# Upgradeable=False (wait for https://issues.redhat.com/browse/OTA-861)
+function wait_upgrade_start(){
+    local retry=0 output
+    echo "Wait for the upgrade to start"
+    while [[ retry -lt 5 ]]; do
+        output="$(oc get clusterversion version -ojson)"
+        if [[ "$(echo $output | jq -r '.status.conditions[] | select(.type == "Progressing").status')" == "True" ]] \
```

Comment:
As an enhancement, I guess we can add one more check point to ensure Progressing=False in `check_migrate_status`

Reply:
I think we already have it in `check_migrate_status`:
```
        if [[ ${avail} == "True" && ${progress} == "False" && ${out} == *"Cluster version is ${SOURCE_VERSION}" ]]; then
            echo -e "Migrate succeed\n\n"
            return 0
        fi
```